src/test/resources/features/convertImage/image.feature

```diff
@@ -0,0 +1,283 @@
+Feature: Image converter
+
+  In order to convert a image file
+  As a valid Converter customer
+
+  Background: Sets authentication
+    Given I set authentication headers
+
+  @functional
+  Scenario: Verify that a image file is converted to another image file with a different extension
+    When I send a POST request to "/convertImage" with the following form data
+      | file             |  @"template/image/image.jpg"      |
+      | md5              |  1589d7f466220b7604858b1eaabdd0d2 |
+      | exportFormat     |  .png                             |
+      | name             |  test                             |
+      | changeSize       |  on                               |
+      | position         |  2,5                              |
+      | height           |  10                               |
+      | width            |  10                               |
+      | gray             |  on                               |
+      | extractMetadata  |  on                               |
+      | extractThumbnail |  on                               |
+    Then I validate the response has the "200" status code
+    And I validate that the response body should match with "common/messageResponse.json" JSON schema
+    And I validate that the response container the following values
+      | status  | 200 |
+
+  @functional
+  Scenario: Verify that the image file is converted to grayscale
```

Comment:
Please consider improving this scenario title to something like: `Verify that an image file is converted to grayscale image`